app.rb

```diff
@@ -1,20 +1,44 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+# Explain how the data is structured
+# What are the properties for each of the two types of hashes
+
+apartments.map{|x| ruby_apartments << Apartment.new(x[:id], x[:address], x[:monthly_rent], x[:square_feet])}
```

Comment:
You could instead do `ruby_apartments = apartments.map{ ... }` so you don't need to worry about pushing inside `.map`

Same deal with `tenants.map{ ... }`